c/zis/services/auth.c

```diff
@@ -90,13 +61,81 @@ static int handleVerifyPassword(AuthServiceParmList *parmList,
 
   FILL_SAF_STATUS(&parmList->safStatus, safRC, racfRC, racfRsn);
   CMS_DEBUG(globalArea, "handleVerifyPassword() done\n");
-  if (idta != NULL) {
-    safeFree(idta, sizeof(IDTA));
-    idta = NULL;
-  }
   return rc;
 }
 
+static int handleGenerateToken(AuthServiceParmList *parmList,
+                                const CrossMemoryServerGlobalArea *globalArea) {
+       if(parmList->safIdtService.safIdtServiceVersion > 1) {
+            return RC_ZIS_AUTHSRV_BAD_SAF_SERVICE_VERSION;
+       }
+    ACEE *acee = NULL;
```

Comment:
Formatting is incorrect. Please refer to the rest of this file to format appropriately.

Reply:
fixed according to the rest of the file